hack/verify-gofmt.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Why we use `/bin/bash` here, but use `/usr/bin/env bash` above?

Based on http://stackoverflow.com/questions/16365130/the-difference-between-usr-bin-env-bash-and-usr-bin-bash, there are pros and cons. We should at least make it consistent.